### ISSUE TITLE
Navigate up and down the list with arrows

### ISSUE THREAD

**Issue by mdupont** [BlueBrain/nexus#152]
_From @kenjinp on October 17, 2018 14:4_

There should be keyboard navigation and a snazzy animation between resource detail views

_Copied from original issue: BlueBrain/nexus-explorer#34_
